.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,55 @@
+<!-- Check the following before filing an issue -->
+> Before filing an issue please check that the issue is not already addressed by the following:
```

Comment:
Trying to convey that it's not actually part of the issue and doesn't need to be included.